store/bsdb/bucket_schema.go

```diff
@@ -61,6 +61,9 @@ type Bucket struct {
 	UpdateTime int64 `gorm:"column:update_time"`
 	// Tags
 	Tags datatypes.JSON `gorm:"column:tags;TYPE:json"` // tags
+	// OffChainStatus represents the status of a bucket in the off-chain storage.
+	// It is used to track the current state of the bucket with respect to off-chain operations,
+	OffChainStatus int `gorm:"column:off_chain_status;type:int"`
```

Comment:
can you leave some examples here to describe what status an given int value can represent?